src/plugins/stats.ts

```diff
@@ -1,9 +1,9 @@
 // Built-in plugin to expose stats about the deployment
-module.exports = async robot => {
+module.exports = async (robot: any): Promise<void> => {
```

Comment:
we should use a type for robot, instead of any

Reply:
Since the `robot` type is probably going to be used everywhere, I'm imagining we'll import it into the plugins once we define it in core. Does that sound like the right way to tackle that?